stable/mission-control/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: mission-control
 description: A Helm chart for JFrog Mission Control
-version: 0.5.2
-appVersion: 3.1.2
+version: 0.5.3
```

Comment:
@amithins Change chart version to 0.6.0 as we have released minor version of JFMC.